zk-index.el

```diff
@@ -159,16 +159,10 @@ Adds zk-id as an Embark target, and adds `zk-id-map' and
 
 (defun zk-index--format-candidates (&optional files format)
   "Return a list of FILES as formatted candidates, following FORMAT.
-
-See `zk--format' for details about FORMAT. If nil,
-`zk-completion-at-point-format' will be used by default.
-
-FILES must be a list of filepaths. If nil, all files in
-`zk-directory' will be returned as formatted candidates."
-  (let* ((zk-index-format (if zk-index-invisible-ids "%t %i"
```

Comment:
This case for `zk-index-invisible-ids` shouldn't be taken out.

Reply:
Any reason for that? Unless I'm missing something, nothing depends on the specific `%t %i` format. All the code that checks for `zk-index-invisible-ids` uses `re-search-forward` to find the link or the ID no matter where it is on the line.